client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/TicketTypeUpdateTable.tsx

```diff
@@ -126,21 +150,37 @@ export const TicketTypeUpdateTable = (props: TicketTypeTableProps) => {
                   key={eventInstanceID + index + 'ticket type price'}
                   className={'px-2'}
                 >
-                  {getTicketTypePrice(
-                    InstanceTicketTypesField.value[index].typeID,
-                    'price',
-                    ticketTypes,
-                  )}
+                  <Field
+                    name={`${InstanceTicketTypesField.name}[${index}].typePrice`}
+                    type={'text'}
+                    component={InputControl}
+                    hidden={true}
+                    label={'Ticket Price'}
+                    id={eventInstanceID}
+                    className={{
+                      controlClass: '',
+                      inputClass: 'w-[150px]',
```

Comment:
Similarly with this and the following field, could you reduce the width to `75px`

Reply:
Changed to width to 75px